public/js/draw/draw.js

```diff
@@ -87,7 +87,7 @@ function makeNode(posX, posY, jsonObj) {
     } else {
         var nodeWidth = 40;
         var nodeHeight = 32;
-        var nodeFill = nodeColourId;
+        var nodeFill = $('#select-colour').css('background-color');
```

Comment:
It seems like here you want to use the colour picker's `value` attribute (that's typically what we do for inputs).